zdm/data/Surveys/parkes_mb_class_I.dat

```diff
@@ -0,0 +1,30 @@
+FBAR  1382 #MHz
```

Comment:
This is a file that needs to change - I used an old version with an FRB in the wrong place and the incorrect Tobs. Will be updated in phase III.